blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -123,6 +127,16 @@ public static void track(HttpRequest request) {
         }
     }
 
+    static String findTid( String uri ) {
+
+        Matcher m = patternGetTid.matcher( uri );
+
+        if( m.matches() )
+            return m.group( 1 );
+        else
+            return null;
```

Comment:
Oh, I read the code wrong... I thought it was tenantId.someMethod() so if tenantId is null, you get NPE. But it's actually tenantId*s*. Never mind... Then I don't think there's a point of returning empty string.